src/test/java/api/loans/LoanAPITests.java

```diff
@@ -1582,6 +1583,27 @@ public void canCreateMultipleLoansWithServicePoints()
 
   }
 
+  @Test
+  public void SingleLoanContainBorrowerInformation()
```

Comment:
Why has this been introduced as a separate test, rather than adding to `canCreateALoan`?